src/CONST.ts

```diff
@@ -924,10 +924,16 @@ const CONST = {
         MERCHANT: 'merchant',
         FROM: 'from',
         TO: 'to',
+        CATEGORY: 'category',
+        TAG: 'tag',
         TOTAL: 'total',
         TYPE: 'type',
         ACTION: 'action',
     },
+    SEARCH_TABLE_OPTIONAL_COLUMNS: {
+        CATEGORY: 'category',
+        TAG: 'tag',
+    },
```

Comment:
I think we can remove this and just use the values defined in `SEARCH_TABLE_COLUMNS`